packages/diagrams/frontend/sirius-components-diagrams/src/converter/convertDiagram.ts

```diff
@@ -219,6 +219,9 @@ export const convertDiagram = (
       kind: gqlDiagram.metadata.kind,
       targetObjectId: gqlDiagram.targetObjectId,
     },
+    style: {
+      background: gqlDiagram.style?.background,
```

Comment:
`gqlDiagram.style` cannot be `null` or `undefined` given its TypeScript definition so this `?.` is unnecessary.